content/en/docs/setup/best-practices/certificates.md

```diff
@@ -22,6 +22,7 @@ This page explains the certificates that your cluster requires.
 Kubernetes requires PKI for the following operations:
 
 * Client certificates for the kubelet to authenticate to the API server
+* Server certificates for the [kubelet endpoint](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#client-and-serving-certificates)
```

Comment:
I'd write:
```markdown
* Server certificates for the kubelet
```

Not sure if these are _required_ or just very strongly recommended.

Reply:
How about just merging the client and server certificates like the front-proxy?

* [Client and server certificates for the kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#client-and-serving-certificates)

Reply:
It's more the “requires” on line 22 that concerns me.

Reply:
I see your point. 

```
* Client certificates for the API server to talk to the kubelets
```

We already have this in the list. If this is required, isn't the server certificate of kubelets also required? I'm not sure if a TLS with only client certificate is possible.

Reply:
The thing I'm not sure about is whether Kubernetes still allows and supports plain HTTP between the API server and the kubelet.

Reply:
@sftim do we feel like your feedback is binding? Do we want to remove the 'requires' word from the previous statement? Open to alternatives too!

Reply:
So long as the final page either gets a tech review from SIG auth, or we're otherwise convinced the wording is accurate, I'm fine with however we do it.

Reply:
I agree. What do you think about adding a sentence after the list that links to the kubelet certificate reference page?